### PR TITLE
chore: add 3P workflow operations

### PR DESCRIPTION
**Pull request by navneel99**
## Summary

Adding 3rd party workflow operations. These operations allow custom nodes in the workflow to be created by 3rd parties.

## Work-item

https://app.devrev.ai/devrev/works/ISS-102170

### CODE REVIEW

File: 14-operations/code/src/fixtures/on_work_created_event.json
Comment (altafDevRev) [inline 14-operations/code/src/fixtures/on_work_created_event.json:1]: Maybe remove this.

File: 14-operations/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    description: function to handle operations
+
+operations:
+  - name: get_temperature
+    display_name: Get Temperature
+    description: Operation to get the temperature of a city
+    slug: get_temperature
+    function: operation_handler
+    type: action
+    inputs:
```
Comment (altafDevRev) [inline 14-operations/manifest.yaml:19]: Maybe we can add some comments explaining what these inputs are and how they are different from snap-in inputs.

**Review by altafDevRev — APPROVED** [review]

File: 14-operations/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Snap-in containing custom operations
+
+The snap-in contains custom operations which can be used in the Workflow Builder once snap-in gets successfully activated.
+
+Operations are workflow node blueprint which are used to create workflow steps(nodes).
```
Comment (codeon) [inline 14-operations/README.md:5]: Let's remove node from here.

File: 14-operations/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Operations in the snap-in
+
+The snap-in contains the following operations:
+
+- Get Temperature: Prints the temperature of city from a map of temperatures in the backend. Shows how to override context in an operation
```
Comment (codeon) [inline 14-operations/README.md:11]:
```suggestion
- Get temperature: Prints the temperature of city from a map of temperatures in the backend. Shows how to override context in an operation
```

File: 14-operations/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Post comment on Ticket: Posts a specific comment only on tickets. Shows the usage of the DevRev SDK.
+
+- Send Slack Message: Posts a message to specified slack channel. Shows the usage of keyrings to connect to external systems.
```
Comment (codeon) [inline 14-operations/README.md:15]:
```suggestion
- Send slack message: Posts a message to specified slack channel. Shows the usage of keyrings to connect to external systems.
```

**Review by codeon — APPROVED** [review]

**Pull request merged**